eth/downloader/queue.go

```diff
@@ -318,7 +318,7 @@ func (q *queue) Schedule(headers []*types.Header, from uint64) []*types.Header {
 			q.blockTaskQueue.Push(header, -int64(header.Number().Uint64()))
 		}
 		// Queue for receipt retrieval
-		if q.mode == FastSync && !header.EmptyReceipts() {
```

Comment:
I don't think this is right. Since `fastSync` is gone, this block should also be gone

Reply:
Removed.